frontend/src/metabase/entities/questions/forms.js

```diff
@@ -16,7 +16,7 @@ const FORM_FIELDS = [
   {
     name: "is_write",
     title: t`Is Write`,
-    description: t`Write questions can be used for experimental actions.`,
+    description: t`Write questions that can be used for experimental actions.`,
```

Comment:
I spoke with @kulyk and this doesn't seem to be a typo. Admittedly, it sounds a bit strange, but in the context of data apps "write questions" is a noun. Adding `that` to this sentence changes its meaning to `Write` (verb) + `questions` (noun)